src/styles/blocks/product-card.scss

```
@@ -0,0 +1,93 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.page {
+  width: 198px;
```

```suggestion
  width: 200px;
```
just add the `box-sizing: border-box;`